cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/TimePartitionedFileSet.java

```diff
@@ -50,6 +50,26 @@
    */
   void addPartition(long time, String path);
 
+  // Why are some of these methods methodNameByTime, while others are just methodName?
```

Comment:
I did not get an answer, but my assumption is that the consistency of the methods was not a concern at the time they were named.
All of the methods act with `long time` as the first parameter, but the suffix is not consistent.
What do you mean keep this comment, though? You mean with an answer to the question?